packages/client/src/views/OfficeHome/readyToPrint/ReadyToPrint.tsx

```diff
@@ -198,6 +203,7 @@ class ReadyToPrintComponent extends React.Component<
           ) => {
             e && e.stopPropagation()
             if (downloadStatus === DOWNLOAD_STATUS.DOWNLOADED) {
+              this.props.clearCorrectionChange(reg.id)
```

Comment:
@sadmananik Could I trouble you to just refactor this action name to a bit more common for our usecase? How about `clearCorrectionAndPrintChanges`?

Reply:
Done